frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -27,6 +27,15 @@ import type { Parameter } from "metabase-types/types/Parameter";
 
 const MAX_SEARCH_RESULTS = 100;
 
+const fieldValuesWidgetPropTypes = {
```

Comment:
Any particular reason / case where it makes sense to use this pattern of `const thingPropTypes = {}` and then assigning it later vs just defining them "inline" as you've done some other files? 
```js
Thing.propTypes = {
   message: PropTypes.string
}
```

No particular preference on my end, I've just traditionally done it the other way across the board.

Reply:
@kdoh, thank you for the review 🙇 

This was a suggestion from Anton for files with more than one component to cover with propTypes.

https://github.com/metabase/metabase/issues/16224#issuecomment-847910883